src/android/PluginError.java

```diff
@@ -2,18 +2,18 @@
 
 public enum PluginError {
 
-    BIOMETRIC_UNKNOWN_ERROR(-100),
-    BIOMETRIC_AUTHENTICATION_FAILED(-102, "Authentication failed"),
-    BIOMETRIC_HARDWARE_NOT_SUPPORTED(-104),
-    BIOMETRIC_NOT_ENROLLED(-106),
-    BIOMETRIC_DISMISSED(-108),
-    BIOMETRIC_PIN_OR_PATTERN_DISMISSED(-109),
+    BIOMETRIC_UNKNOWN_ERROR(-100, "An unknown error occurred."),
+    BIOMETRIC_AUTHENTICATION_FAILED(-102, "Authentication failed."),
+    BIOMETRIC_HARDWARE_NOT_SUPPORTED(-104, "Biometrics is not supported on this hardware."),
+    BIOMETRIC_NOT_ENROLLED(-106, "Biometrics is not configured."),
+    BIOMETRIC_DISMISSED(-108, "Biometrics was dismissed by user."),
```

Comment:
Not super important but maybe this could be "Biometric prompt was dismissed by user."